_posts/2018-10-17-why-ruby-on-rails.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: news
+title: "Why Ruby on Rails?"
+categories: developer
+author: Ngari Ndung'u
+blog-image: rails-blog.jpg
+intro: Ruby on Rails, abbreviated RoR or just Rails is a web application framework for developing database backed web applications.
+  Rails has been used to build some of the most popular websites including Github, Airbnb and Hulu. Twitter also began life as a rails application.
+  While the choice of framework is not a straight forward one, choosing rails looks like a pretty safe bet.
+---
+{:.post-figure}
+![Photo of a railroad with pink vegetation](/assets/images/blog/{{page.blog-image}}){:class="img-responsive center"}
+<span style="font-size:12px; margin:auto; display:block; text-align:center;">Photo by Esteban Trivelli on Unsplash</span>
+
+## What is Rails?
+
+{{ page.intro }}
+
+As the name implies, rails is built on top of ruby. Ruby is a dynamic programming language built for simplicity and speed.
```

Comment:
Please link to the main ruby-lang.org and add a more specific example. That help world is the same for almost every other language. You can find somethings on th ruby website